dpytools/embeds.py

```diff
@@ -118,3 +119,141 @@ def add_fields(self, inline=True, **kwargs):
         """
         for name, value in kwargs.items():
             self.add_field(name=name, value=value, inline=inline)
+
+            
+class EmbedList:
```

Comment:
Overall i like where this class is heading but i thing it's missing some important things. For example, author url and icon_url, or the embed url, etc. The class should be able to make a full Embed. 

A suggestion could be accepting a `base_embed` argument on `__init__` that contains everyting but the fields and just paginate the fields using the base embed as template.